### PR TITLE
Add tests for different Administrate::Fields types

### PR DESCRIPTION
**Pull request by jonatasrancan**

### CODE REVIEW

File: spec/dummy/Gemfile.lock [hunk 1]
```diff
@@ -2,7 +2,7 @@ PATH
   remote: ../..
   specs:
     administrate_exportable (0.1.0)
-      administrate
+      administrate (> 0.10.0)
```
Comment (andreibondarev) [inline spec/dummy/Gemfile.lock]: Is this an intentional change?

File: spec/administrate_exportable/exporter_service_spec.rb [hunk 2]
```diff
@@ -2,23 +2,126 @@
...
+      expect(header).to eq 'Id,First Name,Last Name,Dogs,Cat,Email,Password,Created At,Updated At'
     end
 
-    it 'exports correct data' do
-      user = User.create(first_name: 'John', last_name: 'Doe')
-      user.dogs.create(name: 'Wolf')
-      user.create_cat(name: 'Black Panther')
+    context 'exporting Field::Number' do
+      it 'exports correct data' do
+        data = result.split("\n").last.split(',')[0]
```
Comment (andreibondarev) [inline spec/administrate_exportable/exporter_service_spec.rb]: Should we abstract this `data = result.split("\n").last.split(',')` out?

File: spec/dummy/Gemfile.lock (hunk 1 above)
Comment (jonatasrancan) [reply spec/dummy/Gemfile.lock]: Yeah. Forgot to update the dummy app, when we specified the administrate  version

**Comment by andreibondarev** [comment]
@jonatasrancan Please double-check that you can run the dummy app and run the manual export through the UI.

**Pull request merged**